### PR TITLE
initial fastlane structure

### PR DESCRIPTION
**Pull request by IzzySoft**
As discussed in #100 (and apologies for the delay).

### CODE REVIEW

File: fastlane/metadata/android/en-US/images/icon.png
Comment (pachi81) [inline fastlane/metadata/android/en-US/images/icon.png:1]: Please use the last icon: https://github.com/pachi81/GlucoDataHandler/blob/master/images/GDH_icon.png

File: fastlane/metadata/android/en-US/full_description.txt
Comment (pachi81) [inline fastlane/metadata/android/en-US/full_description.txt:1]:
```suggestion
<p><i>Glucose Data Handler</i> receives glucose values from, LibreLinkUp, <a href='https://github.com/j-kaltes/Juggluco' target='_blank' rel='nofollow noopener'>Juggluco</a> or <a href='https://github.com/NightscoutFoundation/xDrip' target='_blank' rel='nofollow noopener'>xDrip+</a> and provides widgets, notifications, sends glucodata broadcasts to other apps and more. The app also provides support for WearOS devices, Android Auto (using GlucoDataAuto) and Tasker.</p>
```

File: fastlane/metadata/android/en-US/images/phoneScreenshots/03.png
Comment (pachi81) [inline fastlane/metadata/android/en-US/images/phoneScreenshots/03.png:1]: With version 0.9.8, the Android Auto part has moved to an extra app called GlucoDataAuto, as this part is not PlayStore conform.
So maybe you have to remove these screenshots or add also GlucoDataAuto...

File: fastlane/metadata/android/en-US/short_description.txt
Comment (pachi81) [inline fastlane/metadata/android/en-US/short_description.txt:1]:
```suggestion
receives glucose data from from LibreLinkUp, Juggluco or xDrip+.
```

**Review by pachi81 — CHANGES_REQUESTED** [review]

File: fastlane/metadata/android/en-US/images/icon.png
Comment (IzzySoft) [reply fastlane/metadata/android/en-US/images/icon.png:1]: Updated.

File: fastlane/metadata/android/en-US/images/phoneScreenshots/03.png
Comment (IzzySoft) [reply fastlane/metadata/android/en-US/images/phoneScreenshots/03.png:1]: Can I leave this to you, after merge? The idea was I provide you what I have set up on my end as a "starter package" so you have the structures ready and then can update/replace what you see needed :wink:

**Review by pachi81 — APPROVED** [review]

**Comment by IzzySoft** [comment]
Thanks for merging! Activated it now on my end, so future changes will be pulled along with new releases from now on.

Btw, that Google badge looks a bit lonely in your Readme. Maybe you want to [pick a badge](https://gitlab.com/IzzyOnDroid/repo/#if-my-app-is-available-here-do-you-have-a-badge-i-can-use-to-link-to-it) from me to accompany it, linkking to [your app in my repo](https://apt.izzysoft.de/packages/de.michelinside.glucodatahandler/)? :smiley:

**Pull request merged**
